index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
kindly remove all default icons that comes by default with react to make your work look professional